dashboard/app/views/layouts/_header.html.haml

```diff
@@ -12,6 +12,13 @@
 - show_bug_links = (script_level && script.show_report_bug_link?) || (level && level.try(:is_project_level))
 
 .header-wrapper.hide_on_mobile{class: ('feedback-bug-links' if show_bug_links)}
+  - if current_user && script_level
+    #progresswarning
+      %a{href: "http://teacherblog.code.org/", target: "_blank"}
```

Comment:
idk, seems like most people clicking on this link will not be teachers, so teacherblog seems like a bit of a strange place to take them. dont feel super strongly about this